_docs/delegation-dev.md

```diff
@@ -5,13 +5,13 @@ author: charles
 tags: [delegator]
 date: 2024-02-19
 ---
-This guide provides links and instructions for staking by delegating an existing Turing Network collator. See [Collator Overview](https://docs.oak.tech/docs/collators/) for instructions on setting up your own collator.
+This guide provides links and instructions for staking by delegating an existing Turing Network collator. See [Collator Overview](https://docs.avaprotocol.org/docs/collators/) for instructions on setting up your own collator.
```

Comment:
You need to test the target link. All target link needs to work. This is clearly wrong since we don’t have `docs.avaprotocol.org`